core/sqf/monitor/linux/clio.cxx

```diff
@@ -377,6 +379,21 @@ Local_IO_To_Monitor::Local_IO_To_Monitor(int pv_pid)
         tmpptr++;
     }
 
+    // Remove the domain portion of the name if any
+    char str1[MPI_MAX_PROCESSOR_NAME];
```

Comment:
It's a larger buffer than needed, but I should look at all the uses of MPI_MAX_PROCESSOR_NAME. This is targeted for cleanup is MPI usage which is quite minimal in the current code base.